.grit/patterns/mui4_to_mui5.md

```diff
@@ -0,0 +1,27 @@
+## title: Convert Rename theme.palette.type In MUI v4 To MUI v5
+
+This pattern Rename theme.palette.type in MUI v4 to MUI v5
+
+tags: #react, #migration, #complex
+
+```grit
+engine marzano(0.1)
+language js
+
+// Most of the logic for this pattern is in mui4_to_mui5.grit
+// https://github.com/getgrit/js/blob/main/.grit/patterns/mui4_to_mui5.grit
```

Comment:
This file does not exist. Don't link to files that don't exist.

Reply:
This has resolved @morgante